src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier color = new ColorSupplier();
+
+    public String getRandomBall() {
+
+        Random random = new Random();

```

Comment:
lets make Random a class level variable (similar to ColorSupplier) to avoid creating a new instance each time this method is inoked